### PR TITLE
[MGDSTRM-1571] - Return 0 if empty

### PR DESCRIPTION
**Pull request by 1984shekhar**
"OR on() vector(0)" will return 0 if reponse is empty. This will ensure that output 0 retains it's color. ValueMapping for null set color to white.

PR_NotReady

### CODE REVIEW

File: install/resources/monitoring-global/dashboards/fleet.yaml [hunk 1]
```diff
@@ -621,7 +621,7 @@ spec:
           "tableColumn": "",
           "targets": [
             {
-              "expr": " count(strimzi_resource_state{kind=~\"Kafka\"} == 0)",
+              "expr": " count(strimzi_resource_state{kind=~\"Kafka\"} == 0) OR on() vector(0)",
```
Comment (obrienrobert) [inline install/resources/monitoring-global/dashboards/fleet.yaml:624]: Could we remove the single leading white space before count? Just to keep it in line with the other `expr:` fields in the file :relaxed:

**Comment by 1984shekhar** [comment]
Yes removed the space.

**Comment by obrienrobert** [comment]
Tested the expression by placing it into the `Total Kafka Clusters (Not Ready)` Observtorium Grafana panel. The expression returned `0`. 

clusters_not_ready

The original expression returned `3`.

clusters_not_ready_3

Discussed with Chandra who is looking into it.

**Comment by obrienrobert** [comment]
Tested this again today with @1984shekhar. All looks good. The issue in the previous comment looked to be a refresh/timing discrepancy. 

LGTM.

**Review by obrienrobert — APPROVED** [review]

**Comment by obrienrobert** [comment]
Updated GrafanaDashboard CR. Verified that the changes are present in the Kafka Fleet Panel:

total_kafka_clusters_not_ready

**Review by obrienrobert — APPROVED** [review]

**Comment by obrienrobert** [comment]
@david-martin I've verified and made the changes to the Fleetwide dashboard. Is this OK to merge?

**Review by david-martin — APPROVED** [review]

**Pull request merged**